Makefile

```diff
@@ -99,4 +102,4 @@ build/docker/push:
 build/docker/untag:
 	echo "docker rmi brigadehub/$(PACKAGENAME):release"
 
-.PHONY: start lint test db install build link
+.PHONY: start lint test db install build link .env
```

Comment:
http://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile